L/LibGit2/bundled/patches/libgit2-win32-ownership.patch

```diff
@@ -0,0 +1,44 @@
+From cdff2f0237f663e0f68155655a8b66d05c1ec716 Mon Sep 17 00:00:00 2001
+From: Edward Thomson <ethomson@edwardthomson.com>
+Date: Mon, 13 Jun 2022 21:34:01 -0400
+Subject: [PATCH] repo: allow administrator to own the configuration
+
+Update our ownership checks that were introduced in libgit2 v1.4.3
+(to combat CVE 2022-24765). These were not compatible with git's; git
+itself allows administrators to own the path. Our checks now match
+this behavior.
+---
+ src/libgit2/repository.c  | 2 +-
+ tests/libgit2/repo/open.c | 5 +++--
+ 2 files changed, 4 insertions(+), 3 deletions(-)
+
+diff --git a/src/libgit2/repository.c b/src/libgit2/repository.c
+index 48a0b70f519..d2484318f10 100644
+--- a/src/libgit2/repository.c
++++ b/src/libgit2/repository.c
```

Comment:
In v1.4.3 this file was called `src/repository.c`: https://github.com/libgit2/libgit2/blob/v1.4.3/src/repository.c#L515.  If we're lucky, we only need to rename this, the context looks the same.  Similarly for the test file, which was called `tests/repo/open.c` (we can also discard that hunk, we don't run the tests here)